data/aliases.ts

```diff
@@ -1032,7 +1032,7 @@ export const BattleAliases: {[alias: string]: string} = {
 	yadoran: "Slowbro",
 	koiru: "Magnemite",
 	reakoiru: "Magneton",
-	kamonegi: "Farfetch'd",
+	kamonegi: "Farfetch\u2019d",
```

Comment:
I purposefully skipped this because gets toID-ed anyway (https://github.com/smogon/pokemon-showdown/blob/master/sim/dex.ts#L354)

Reply:
I swapped back.